frontend/www/js/omegaup/components/contest/Groupsv2.vue

```diff
@@ -56,16 +56,16 @@ import Autocomplete from '../Autocomplete.vue';
   },
 })
 export default class Groups extends Vue {
-  @Prop() initialGroups!: types.ContestGroup[];
+  @Prop() data!: types.ContestGroup[];
```

Comment:
```suggestion
  @Prop() groups!: types.ContestGroup[];
```

así el componente padre puede hacer nada más `:groups="groups"`.